modules/fancyBox/source/jquery.fancybox.js

```diff
@@ -1865,7 +1867,36 @@
 				return;
 			}
 
-			title = $('<div class="fancybox-title fancybox-title-' + type + '-wrap">' + text + '</div>');
+			if (['inside', 'outside', 'over'].includes(type)) {
+				var parts = [text, link].filter(Boolean);
+				title = $('<div class="fancybox-title fancybox-title-' + type + '-wrap">' + parts.join(sep) + '</div>');
+			}
+			else { // 'float'
+				title = $('<div class="fancybox-title fancybox-title-' + type + '-wrap">' + text + '</div>');
+
+				title.appendTo('body');
+
+				if (IE) {
```

Comment:
Ik weet dat je dit hebt overgenomen maar dat kan wel weg, zeker als je hieronder template literals gebruikt die IE niet ondersteunt.

Reply:
Keeping it for now since we don't introduce non-IE compatible syntax anymore re the next comment.